cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -343,19 +344,20 @@ script "js_ri_cleanup", type: "javascript" do
     _.each(ri_recos, function(recos){
       var reco_details = recos["recommendationDetails"];
       _.each(reco_details, function(reco){
+        var platform
```

Comment:
Is this supposed to be here? It doesn't appear to have a value.

Reply:
You were right! Deleted that var